site/index.md

```diff
@@ -21,6 +21,20 @@ Following modules are available:
   - `fs2-data-cbor`: [![Maven Central](https://img.shields.io/maven-central/v/org.gnieh/fs2-data-cbor_2.13.svg)](https://mvnrepository.com/artifact/org.gnieh/fs2-data-cbor_2.13) CBOR parser and trasformation
   - `fs2-data-cbor-json`: [![Maven Central](https://img.shields.io/maven-central/v/org.gnieh/fs2-data-cbor-json_2.13.svg)](https://mvnrepository.com/artifact/org.gnieh/fs2-data-cbor-json_2.13) CBOR/JSON interoperability library
 
+## Adopters
+
+Here's a (non-exhaustive) list of companies and projects that use `fs2-data`. Don't see yours? You can [add it in a PR][add-adopter]!
+
+### Projects
+
+ - [typelevel-toolkit](https://typelevel.org/toolkit/) - A toolkit of great libraries to start building Typelevel apps on JVM, Node.js, and Native!
+ - [http4s-fs2-data](https://http4s.github.io/http4s-fs2-data/) - Provides a set of integration libraries that integrate http4s with the streaming parsers offered by fs2-data.
+ - [smithy4s](https://disneystreaming.github.io/smithy4s/) - An interface definition language (IDL) provided by AWS.
+ - [lichess.org](https://lichess.org/) - A free/libre, open-source chess server powered by volunteers and donations.
```

Comment:
It's an integration library "only" btw, apparently not the core. And let's wait for an OK for mentioning them.

Reply:
I can remove it from now. Everyone is free to add their project or company once it is bootstrapped.